deploy/setup-project.sh

```diff
@@ -91,6 +93,10 @@ do
   gcloud projects add-iam-policy-binding "${PROJECT}" --member serviceAccount:"${IAM_NAME}" --role "${role}"
 done
 
+# Authorize GCE to encrypt/decrypt using Cloud KMS encryption keys.
+# https://cloud.google.com/compute/docs/disks/customer-managed-encryption#before_you_begin
+gcloud projects add-iam-policy-binding "${PROJECT}" --member serviceAccount:"service-${PROJECT_NUMBER}@compute-system.iam.gserviceaccount.com" --role "roles/cloudkms.cryptoKeyEncrypterDecrypter"
+
```

Comment:
this script is for user to set up their cluster, if they do not use cmek feature, this part is not needed. 
Is it better to give an option? If user enable this option, then set it up?

Reply:
That's not a bad point. My thought was that it's better to have this stuff on by default as, eg for testing it makes everything more consistent.

But maybe we don't want to add policy bindings by default.

Reply:
Added a parameter called `ENABLE_KMS`.

Reply:
/lgtm

I think it's good to have to explicitly specify KMS, that's better than having a default that won't be noticed until it's too late.